server/src/main/java/com/lunchtime/LunchTimeApplication.java

```diff
@@ -1,21 +1,13 @@
 package com.lunchtime;
 
-//TODO remove unused imports. You can use Ctrl+Alt+O in Intellij before committing changes
-import com.lunchtime.models.Restaurant;
-import com.lunchtime.repository.RestaurantRepository;
-import com.lunchtime.service.RestaurantService;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
-import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;
 import org.springframework.context.annotation.Bean;
-import org.springframework.context.event.ContextRefreshedEvent;
-import org.springframework.context.event.EventListener;
 import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.ResourceHandlerRegistry;
 import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
-@SpringBootApplication(
-    //TODO One time will be enough
-    exclude = {SecurityAutoConfiguration.class, SecurityAutoConfiguration.class})
```

Comment:
why you delete all line of auto configuration spring security ?

Reply:
Ok, i will return this code. I thought we were turning off security auto config in other place.